src/javascript/app/common/guide.js

```diff
@@ -87,8 +89,13 @@ const Guide = (() => {
     const setEvents = () => {
         $(`${opt.guideBtnID} strong`).click(() => {
             const enjoyhint_instance = new EnjoyHint({});
+            const contractList = $(opt.contractList);
```

Comment:
Can we use snake_case for these constants? `contract_list` and `close_confirmation`

Reply:
since guideBtnID is in the camelCase format, i keep it the same

Reply:
My suggestion would be to make all three consistent like `guide_btn_id`, `contract_list_id`, and `close_confirmation_id`. Do you think this would be better?

Reply:
yeah, right, will handle that, thanks